README.md

```diff
@@ -139,13 +139,29 @@ quicktype pokedex.ts -o src/ios/models.swift
 
 `quicktype` is implemented in TypeScript and requires `nodejs` and `npm` to build and run.
 
+First, install `typescript` globally via `npm`:
```

Comment:
Is this only for the Windows instructions below?  If so, please move it there.

Reply:
I'm not sure. I think it might be a general requirement, but it's a onetime install. It's also possible that the linux scripts do the install, though.

Reply:
`typescript` is a dependency of quicktype in `package.json`, so as soon as you do `npm install` it'll be installed, and everything else you do via `npm` will use that TypeScript.  So I'm pretty sure it's specific to your Windows instructions, since they run `tsc` manually.